src/Endpoints/class-graphql-metadata.php

```diff
@@ -125,7 +125,12 @@ private function register_object_types(): void {
 	private function register_fields(): void {
 		$resolve = function ( \WPGraphQL\Model\Post $graphql_post ) {
 			$post_id = $graphql_post->ID;
-			$post    = WP_Post::get_instance( $post_id );
+
+			if ( ! is_int( $post_id ) ) {
+				return null;
```

Comment:
`register_fields()` has a `void` return type, so it may also make sense to update these returns to match.

```suggestion
				return;
```

Reply:
@alecgeatches, I went ahead and "fixed" this, but then got a SonarCloud complaint and also realized that this is the `$resolve` variable, which is holding a function result (null or array). So I reverted.

Let me know if this makes sense, or if you think there's something I might be missing.

Reply:
Fine with me, whatever works here!